### ISSUE TITLE
Improve contrast of code highlighting

### ISSUE THREAD

**Issue by fonsp** [fonsp/Pluto.jl#2921]
Our code does not always have good readable contrast. 

For example, strings like `md"""` have low contrast, Chrome also points this out:

<img width="491" alt="image" src="https://github.com/fonsp/Pluto.jl/assets/6933510/9d7eebeb-7f2f-475b-9fd3-6ccc3a50c406">

@savq Offered to investigate this (which scoring function should be used for contrast?) and maybe make a new color theme for us! It might be fun to release it together with https://github.com/JuliaPluto/lezer-julia/pull/7 so that things "look new"

**Comment by savq** [comment]
To calculate color contrast accurately we should use the [APCA/WCAG3 lightness contrast function](https://www.myndex.com/APCA/).

The current Web Content Accessibility Guidelines v2 (WCAG2) _contrast ratio_ has several problems. The most notable one is that it overestimates the contrast of dark colors, so a dark theme with colors that pass WCAG2 might actually be unreadable.

Besides following WCAG3, we should also consider using a perceptually uniform color space like [oklab/oklch](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/oklch). Since it's much easier to choose colors that have a nice contrast. These color spaces are included in CSS colors level 4 (most browsers support it since early 2023).

For reference here are some examples of current lightness contrast against the normal cell background `#FEFCF5`:

```css
--cm-comment-color:  #E96BA8; /* Lc = 53.5 */
--cm-number-color:   #815BA4; /* Lc = 74.3 */
--cm-property-color: #B67A48; /* Lc = 61.4 */
--cm-keyword-color:  #EF6155; /* Lc = 56.9 */
--cm-string-color:   #DA5616; /* Lc = 64.1 */
--cm-var-color:      #5668A4; /* Lc = 74.8 */
--cm-macro-color:    #5C8C5F; /* Lc = 64.6 */
```

The passing grade (like the WCAG2 ratio 4.5:1) is Lc 75. 😬


Most colors can be adjusted by lowering the lightness. However, the colors for keywords and strings will be harder to differentiate (they're already pretty close reddish colors), so it might be a good idea to use an entirely different color for strings (I'd personally prefer a green/blueish color.)

**Comment by fonsp** [comment]
That's a cool tool! I think your screenshot in https://github.com/JuliaPluto/lezer-julia/pull/7#issuecomment-2111063451 looks really nice, I'm happy to just go with that! I like the new colors!

**Comment by fonsp** [comment]
hey @savq! could you make a pr with your colors? we can release it together with the new parser
